Framework/Core/include/Framework/HistogramRegistry.h

```diff
@@ -637,7 +637,7 @@ class HistogramRegistry
         return imask(j + i);
       }
     }
-    throw runtime_error("No matching histogram found in HistogramRegistry!");
+    throw runtime_error(Form("No matching histogram %s found in HistogramRegistry!", name));
```

Comment:
No need to use `Form()`, `o2::framework::runtime_error_f()` takes care of it.

```C++
o2::framework::runtime_error_f("No matching histogram %s found in HistogramRegistry!", name);
```

Reply:
hi, @njacazio if you agree I can include this in my next commit for the registry (which will also affect / move this function and therefore would have conflicts with your changes).

Reply:
@ktf thank you, indeed it was a temporary patch ! 
@mkruegerGitHub it's up to you, I just need a way to know which plot I'm missing in the filling

Reply:
indeed it is a good idea to provide this information. I will add it.

Reply:
Ok, I removed the commit